### PR TITLE
Update node.tsx to add a delay for typsetting

### PR DESCRIPTION
**Pull request by wadethestealth**
This is a temporary fix to #4461 and has already been discussed between @rgbkrk and I inside #4461.

### CODE REVIEW

**Review by captainsafia — APPROVED** [review]
Change seems reasonable.

Is the 10ms the only value that will work or is it any value within a certain threshold?

Also, can you add a comment in this change to explain why it was made?

**Comment by wadethestealth** [comment]
@captainsafia it seems that even if I set the value to 0 the extra overhead seems to still make it work, but if I remove the set timeout it goes back to only rendering the first node. The intuition behind the change is that Mathjax is supposedly async, and hydrogen was only ever able to render the first node ran, and it always left the first node in the processing state even though it was finished rendering. So I randomly tried adding a timeout so that I could easier debug and the issue went away completely. Again something wrong is happening here without the setTimeout, but I have tried to step by step debug it multiple times with beautified code, and I have seen no obvious errors. Again this is just a temporary fix to the issue and the real issue is written very in depth inside #4461

File: packages/mathjax/src/node.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ class MathJaxNode_ extends React.Component<Props & MathJaxContextValue> {
    * Render the math once the node is mounted
    */
   componentDidMount() {
-    this.typeset();
+    setTimeout(() => this.typeset(), 10);
```
Comment (rgbkrk) [inline packages/mathjax/src/node.tsx]: I wonder if this would even work with `setTimeout(fn, 0);`, since it still lets some DOM rendering catch up.

**Review by rgbkrk — APPROVED** [review]

File: packages/mathjax/src/node.tsx (hunk 1 above)
Comment (wadethestealth) [reply packages/mathjax/src/node.tsx]: at least for hydrogen setting it to 0 still works, idk if we should leave it as 10 though for tolerance or not

Comment (rgbkrk) [reply packages/mathjax/src/node.tsx]: I'd probably go with 0 if it works as it basically just schedules the work after everything else in the queue. Background: https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful

**Comment by wadethestealth** [comment]
Ready for merge? @rgbkrk and @captainsafia

**Comment by rgbkrk** [comment]
🎉

**Pull request merged**